### PR TITLE
10 array code challenge problems with examples and solutions added

### PR DESCRIPTION
**Pull request by isadominguez314**

### CODE REVIEW

**Comment by isadominguez314** [comment]
i added problems 11-20 (18-20 are a little harder) to the array code practice

File: code-challenges/arrays.md [hunk 1]
```diff
@@ -156,3 +156,144 @@ same_first_last([1, 2, 1]) → true
...
+
+find_odds([1,2,3,4,5,6]) → [1,3,5]
+find_odds([7,1,9,8,5,2,4,6]) → [7,1,9,5]
+find_odds([2,4,6,7]) → [7]
+
+Sample Solution: 
+
+function find_odds(arr) {
+   return arr.filter((num) => num % 2 === 1);
+}
```
Comment (allejo) [inline code-challenges/arrays.md:172]:
~~~suggestion
Given an array of ints, return a new array containing only the odd numbers in the same order that they appeared in the original array.

```
find_odds([1,2,3,4,5,6]) → [1,3,5]
find_odds([7,1,9,8,5,2,4,6]) → [7,1,9,5]
find_odds([2,4,6,7]) → [7]
```

Sample Solution: 

```javascript
function find_odds(arr) {
   return arr.filter((num) => num % 2 === 1);
}
```
~~~

**Review by allejo — CHANGES_REQUESTED** [review]
I know the other examples in this document don't follow this format but since you're touching this file, could you please format the formatting of the questions/answers to this format so that it reads better when the markdown is rendered?

**Comment by isadominguez314** [comment]
> I know the other examples in this document don't follow this format but since you're touching this file, could you please format the formatting of the questions/answers to this format so that it reads better when the markdown is rendered?

yes I can work on that now!

**Comment by isadominguez314** [comment]
just finished the formatting changes!

**Pull request merged**